test/Microsoft.Dnx.Runtime.Tests/PathUtilityFacts.cs

```diff
@@ -69,7 +69,8 @@ public static IEnumerable<object[]> FolderPaths
                         new object[] { "", "C:/folder/app", "C:/folder/app" },
                         new object[] { "app/", "C:/folder/app", "C:/folder/app/" },
                         new object[] { "../diff/app/", "C:/folder/app", "C:/diff/app/" },
-                        new object[] { "../folder1/file", "C:\\folder\\file", "C:\\folder1\\file" }
+                        new object[] { "../folder1/file", "C:\\folder\\file", "C:\\folder1\\file" },
```

Comment:
Can you add?
1. `"C:\\", "C:\\folder1\\file"`
2. `C:\\folder1\\file", "C:\\" `